### ISSUE TITLE
docs: HEAD points to 1.0.5 instead of Master

### ISSUE THREAD

**Issue by topherfangio** [angular/material#7247]
Currently, the HEAD docs site is loading the 1.0.5 version of our JS/CSS files instead of master. It appears to be related to the following output from our Travis build:

```bash
48.52s$ ./scripts/travis-build-init.sh --sha=$TRAVIS_COMMIT
-- Building release...
[17:07:14] Using gulpfile ~/build/angular/material/gulpfile.js
[17:07:14] Starting 'build-scss'...
[17:07:14] Building css files...
[17:07:14] Starting 'build-js'...
[17:07:14] building js files...
[17:07:19] Finished 'build-scss' after 4.92 s
[17:07:19] Finished 'build-js' after 4.88 s
[17:07:19] Starting 'build'...
[17:07:19] Finished 'build' after 17 μs
[17:07:20] Using gulpfile ~/build/angular/material/gulpfile.js
[17:07:20] Starting 'build-all-modules'...
[17:07:20] Building material.core ...
[17:07:20] Building material.core ...
...
... // Output removed to make it shorter
...
[17:07:59] Finished 'build-all-modules' after 18 s
-- Cloning bower-material...
Cloning into 'bower-material'...
remote: Counting objects: 919, done.
remote: Compressing objects: 100% (426/426), done.
remote: Total 919 (delta 548), reused 676 (delta 468), pack-reused 0
Receiving objects: 100% (919/919), 1.02 MiB | 0 bytes/s, done.
Resolving deltas: 100% (548/548), done.
Checking connectivity... done.
-- Copying in build files...
-- Committing and tagging...
[master 61ffe35] release: version 1.0.5-master-1d8fc16
 456 files changed, 873 insertions(+), 682 deletions(-)
 rewrite angular-material.min.css (89%)
 rewrite modules/closure/button/button.min.css (68%)
 rewrite modules/closure/card/card.min.css (69%)
 rewrite modules/closure/tabs/tabs.min.css (85%)
 rewrite modules/closure/toast/toast.min.css (93%)
 rewrite modules/js/button/button.min.css (68%)
 rewrite modules/js/card/card.min.css (69%)
 rewrite modules/js/tabs/tabs.min.css (85%)
 rewrite modules/js/toast/toast.min.css (93%)
-- Pushing to bower-material...
remote: Invalid username or password.
fatal: Authentication failed for 'https://angular:@github.com/angular/bower-material/'
Done. Your build exited with 0.
```

Of note is the second to last line:

```bash
fatal: Authentication failed for 'https://angular:@github.com/angular/bower-material/'
```

@ThomasBurleson Can you take a look at why we might have the wrong credentials for bower-material? Or is this something Jeremy/Rob will need to do?

**Comment by zargold** [comment]
me@MBP[1935]$ bower i --save bower-material
bower ENOTFOUND     Package bower-material not found
 raz@MBP[1935]$ bower i --save bower-material --update
bower ENOTFOUND     Package bower-material not found

**Comment by DevVersion** [comment]
@ThomasBurleson There're still credential errors on the CI for the latest build.

<img width="1014" alt="screen shot 2016-02-23 at 22 26 43" src="https://cloud.githubusercontent.com/assets/4987015/13267283/c81f5dae-da7c-11e5-9eb6-0c08217cca32.png">

**Comment by ThomasBurleson** [comment]
Fixed with SHA 30b370d390765e5da5f24761433586a883a6fbe3
